content/bn/docs/reference/glossary/node.md

```diff
@@ -0,0 +1,18 @@
+---
+title: নোড
+id: node
+date: 2018-04-12
+full_link: /docs/concepts/architecture/nodes/
```

Comment:
/bn

Reply:
That advice looks wrong @asem-hamid 

It would be correct if https://github.com/kubernetes/website/blob/dev-1.24-bn.1/content/bn/docs/concepts/architecture/nodes.md already existed, but it doesn't.